### PR TITLE
Flag for multi base subsitutions

### PR DESCRIPTION
**Pull request by mbiokyle29**
## Description

It has been determined that for certain use cases, it may be desirable to treat a sequence of mismatches independently, vs collapsing them into a `delins`. This is especially the case when the `delins` would be "balanced", in other words 3 deleted --> 3 inserted. This goes against the HGVS spec, so I have added it as an explicit opt-in flag. Note that with the default params, it is unlikely to get a raw alignment that looks like this (since opening a gap is cheaper). In almost all cases, custom alignment params would be required to trigger this case (and the flag needs to be passed). To simplify testing of this, I also added the ability to configure alignment params from the CLI.

## Relevant Links

N/A

### Screenshots & Media

Base alignment:
Screen Shot 2024-06-12 at 2 08 32 PM

With flag but no params:
Screen Shot 2024-06-12 at 2 08 53 PM

With flag and mismatch bias params:
Screen Shot 2024-06-12 at 2 09 09 PM

### Testing

- Unit test added
- Tested toy example through CLI, see images

### Code Flow

N/A

### Edge cases / Breaking Changes / Known Issues

N/A

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.0.4
```
Comment (mbiokyle29) [inline .bumpversion.cfg:2]: Not sure how this got out of sync, but I included the bump to 0.0.6 in this PR

File: palamedes/align.py [hunk 2]
```diff
@@ -106,11 +106,30 @@ def merge_variant_blocks(left: VariantBlock, right: VariantBlock) -> VariantBloc
...
+
+    # extra check for flag, merge if false OR merge if true and left and right
+    # are not single bp mismatches. If true and both are single base mismatches
+    # then we do not merge, this 'splitting'
+    mismatch_split_can_merge = not split_consecutive_mismatches or all(
+        [
+            peek_block.alignment_block.bases != VARIANT_BASE_MISMATCH,
+            next_block.alignment_block.bases != VARIANT_BASE_MISMATCH,
+        ]
+    )
```
Comment (mbiokyle29) [inline palamedes/align.py:130]: Logic is a bit gnarly here, but basically you check if they could be merged using the default logic, and then if the flag is passed, you see if they are both single mismatches. If they are, then you dont merge.

**Review by matandro — APPROVED** [review]
Very elegant way to support this use case.
